test/danetest.c

```diff
@@ -182,18 +159,16 @@ static char *read_to_eol(BIO *f)
         return NULL;
 
     n = strlen(buf);
-
-    if (buf[n-1] != '\n') {
-        if (n+1 == sizeof(buf)) {
-            fprintf(stderr, "%s: warning: input too long\n", progname);
-        } else {
-            fprintf(stderr, "%s: warning: EOF before newline\n", progname);
-        }
+    if (buf[n - 1] != '\n') {
+        if (n + 1 == sizeof(buf))
+            TEST_error("input too long");
+        else
+            TEST_error("EOF before newline");
```

Comment:
because these callstack doesn't propagate up as errors.  it is just EOF in the while loop.